src/parameters.toml

```diff
@@ -763,3 +763,221 @@ alias = "avogad"
+explanation = "Ratio of turbulence kinetic energy to squared friction velocity in the surface layer for the EDMF mixing length closure; denoted κ_*². See Lopez-Gomez et al. (2020) [https://doi.org/10.1029/2020MS002162], Table 1. Note: the square root, i.e. κ_*, is listed in the reference."
+
+[mixing_length_Prandtl_number_scale]
+alias = "Prandtl_number_scale"
+value = 4.076923076923077
+type = "float"
+explanation = "Cospectral budget factor for turbulent Prandtl number for the EDMF mixing length closure, denoted ω_pr. In Lopez-Gomez et al. (2020) [https://doi.org/10.1029/2020MS002162], Eq. 36, it is described as a phenomenological constant, denoted by ω₂ and set to 53/13 ≈ 4.0769..."
+
+[mixing_length_Prandtl_number_0]
+alias = "Prandtl_number_0"
+value = 0.74
+type = "float"
+explanation = "Turbulent Prandtl number in neutral conditions; denoted Pr_{t,0}. See Lopez-Gomez et al. (2020) [https://doi.org/10.1029/2020MS002162], Table 1 and Eq 36."
+
+[mixing_length_Ri_crit]
+alias = "Ri_crit"
+value = 0.25
+type = "float"
+explanation = "Critical gradient Richardson number. It is an upper limit to the gradient Richardson number . See Li (2019) [https://doi.org/10.1016/j.atmosres.2018.09.015], Section 6.2 for details"
+
+[mixing_length_smin_ub]
+alias = "smin_ub"
+value = 0.1
+type = "float"
+
+[mixing_length_smin_rm]
+alias = "smin_rm"
+value = 1.5
+type = "float"
+
+[mixing_length_l_max]
+alias = "l_max"
+value = 1.0e6
+type = "float"
+
+[entrainment_factor]
+alias = "entrainment_factor"
+value = 0.13
+type = "float"
+
+[detrainment_factor]
+alias = "detrainment_factor"
+value = 0.51
+type = "float"
+
+[entrainment_massflux_div_factor]
+alias = "entrainment_massflux_div_factor"
+value = 0.0
+type = "float"
+
+[turbulent_entrainment_factor]
+alias = "turbulent_entrainment_factor"
+value = 0.075
+type = "float"
+
+[entrainment_smin_tke_coeff]
+alias = "entrainment_smin_tke_coeff"
+value = 0.3
+type = "float"
+
+[updraft_mixing_frac]
+alias = "updraft_mixing_frac"
+value = 0.25
+type = "float"
+
+[entrainment_area_limiter_scale]
+alias = "area_limiter_scale"
+value = 10.0
+type = "float"
+
+[entrainment_area_limiter_power]
+alias = "area_limiter_power"
+value = 4.0
+type = "float"
+
+[entrainment_scale]
+alias = "entrainment_scale"
+value = 0.0004
+type = "float"
+
+[entrainment_sorting_power]
+alias = "sorting_power"
+value = 2.0
+type = "float"
+
+[minimum_updraft_velocity]
+alias = "min_upd_velocity"
+value = 0.001
+type = "float"
+
+[minimum_updraft_top]
+alias = "min_updraft_top"
+value = 500.0
+type = "float"
+
+[pressure_normalmode_buoy_coeff1]
+alias = "pressure_normalmode_buoy_coeff1"
+value = 0.12
+type = "float"
+
+[pressure_normalmode_buoy_coeff2]
+alias = "pressure_normalmode_buoy_coeff2"
+value = 0.0
+type = "float"
+
+[pressure_normalmode_adv_coeff]
+alias = "pressure_normalmode_adv_coeff"
+value = 0.1
+type = "float"
+
+[pressure_normalmode_drag_coeff]
+alias = "pressure_normalmode_drag_coeff"
+value = 10.0
+type = "float"
```

Comment:
@jiahe23 - do you remember what this one is?

Reply:
same